smartsim/_core/generation/generator.py

```diff
@@ -136,10 +145,22 @@ def generate_job(self, job: Job, job_index: int) -> pathlib.Path:
             dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
             log_file.write(f"Generation start date and time: {dt_string}\n")
 
+        # Create output files
+        out_file, err_file = self._output_files(log_path, job.entity.name)
+        # Open and write to .out file
+        with open(out_file, mode="w", encoding="utf-8") as log_file:
+            dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
+            log_file.write(f"Generation start date and time: {dt_string}\n")
+
+        # Open and write to .err file
+        with open(err_file, mode="w", encoding="utf-8") as log_file:
+            dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
+            log_file.write(f"Generation start date and time: {dt_string}\n")
```

Comment:
If you want, I would actually consider removing these generation messages from these files. In the past they were reserved for the raw stdout and stderr streams from the application itself without any messages from SmartSim

Reply:
Depending on how the launchers work in the backend, some may even end up just overwriting the file (iirc the `ShellLauncher` would currently do that).